packages/content/tests/Application/ExampleTestBundle/Link/ExampleLinkProvider.php

```diff
@@ -60,7 +60,7 @@ public function preload(array $hrefs, string $locale, bool $published = true): i
         $result = [];
         foreach ($examples as $example) {
             $dimensionContent = $this->contentManager->resolve($example, $dimensionAttributes);
-            $this->exampleReferenceStore->add($example->getId());
+            $this->exampleReferenceStore->add((string) $example->getId(), Example::RESOURCE_KEY);
```

Comment:
can we rename this properties as its now always one instance

```suggestion
            $this->referenceStore->add((string) $example->getId(), Example::RESOURCE_KEY);
```